### ISSUE TITLE
[BUG] Nextcloud Hub 7 (28.0.1) slow performance

### ISSUE THREAD

**Issue by simpleacc** [linuxserver/docker-nextcloud#412]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am trying to run Nextcloud (Docker) but it is extremely slow, it is empty, does not have files yet. For example, it needs 28s to load the profile page.

### Expected Behavior

It should go faster or so

### Steps To Reproduce

1 - create the following docker-compose.yml

```
version: "3"
services:
    nextcloud:
        image: lscr.io/linuxserver/nextcloud:latest
        container_name: NEXTCLOUD
        environment:
            - TZ=Etc/UTC
        volumes:
            - ./config:/config
            - ./data:/data
        ports:
            - 443:443
        restart: unless-stopped
        networks:
            VLAN14:
                ipv4_address: 10.0.14.100
networks:
    VLAN14:
        external: true
```

2 - Configure Nextcloud DB connection with the browser and http://10.0.14.100
3 - Wait, wait, wait and wait until the pages load.

### Environment

```markdown
- OS Debian 12 (bookworm)
- CPU: Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (8 cores)
- Memory: 32GB RAM
- Network: 500mbps (upload/download)
- Docker installation: apt install docker docker-compose -y
```

### CPU architecture

x86-64

### Docker creation

```bash
version: "3"
services:
    nextcloud:
        image: lscr.io/linuxserver/nextcloud:latest
        container_name: NEXTCLOUD
        environment:
            - TZ=Etc/UTC
        volumes:
            - ./config:/config
            - ./data:/data
        ports:
            - 443:443
        restart: unless-stopped
        networks:
            VLAN14:
                ipv4_address: 10.0.14.100
networks:
    VLAN14:
        external: true
```

### Container logs

```bash
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] 02-default-location: skipped
[migrations] done
usermod: no changes
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    911
User GID:    911
───────────────────────────────────────

using keys found in /config/keys
Initializing nextcloud 28.0.1.1 (this can take a while) ...
Setting permissions
Initializing finished
[custom-init] No custom files found, skipping...
[ls.io-init] done.
```

**Comment by simpleacc** [comment]
Maybe I have missed something in the configuration side? Like php-local.ini,  www2.conf, nginx.conf or default.conf?

**Comment by Dexus** [comment]
no same here, looks like we have here a problem with php or nextcloud at all...

**Comment by carlinux** [comment]
I'm experiencing the same, after the upgrade to 28 it is somehow laggy, no matter what cache or config I try. And some icons look bad and the theme colours are off.. not sure if it's related or related to the container at all but .. It is happening.

**Comment by Dexus** [comment]
This should be fixed via Nextcloud Server 28.0.3.

**Comment by homerr** [comment]
As we're now on 28.0.4 I'm closing this off.
